src/main/java/crystal/Storage.java

```diff
@@ -0,0 +1,200 @@
+package crystal;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+import crystal.task.Task;
+import crystal.task.Todo;
+import crystal.task.Deadline;
+import crystal.task.Event;
+
+/**
+ * Represents the Storage task.
+ *
+ */
+
+public class Storage {
+    String filepath;
+
+    /**
+     * Constructor for Storage class.
+     *
+     * @param filepath filepath where the list is stored at.
+     *
+     */
+
+    public Storage(String filepath) {
+        this.filepath = filepath;
+    }
+
+    /**
+     * Method to write to the file.
+     *
+     * @param filepath Filepath to store the list at
+     * @param textToAdd Text to be printed on the file
+     *
+     */
+    //Save to file
+    public void writeToFile(String filepath,String textToAdd) throws IOException {
+        FileWriter fw = new FileWriter(filepath);
+        fw.write(textToAdd);
+        fw.close();
+    }
+
+    /**
+     * Method to save the file.
+     *
+     * @param tasks Tasklist to be saved
+     *
+     */
+    public void saveFile(TaskList tasks) {
+        StringBuilder str = new StringBuilder();
+        for (int i = 0; i < tasks.size(); i++) {
+
+            if (tasks.get(i).toString().startsWith("[T]")) {
```

Comment:
Think abt abstracting the task representation in the storage file, inside the Task object, this way, if you have more tasks in the future, you do not need to touch the storage.java file.